src/cmd/cli/command/commands.go

```diff
@@ -582,9 +587,9 @@ var generateCmd = &cobra.Command{
 			cd = "`cd " + prompt.Folder + "` and "
 		}
 
-		// Load the project and check for empty environment variables
-		loader := compose.Loader{ComposeFilePath: filepath.Join(prompt.Folder, "compose.yaml")}
-		project, _ := loader.LoadCompose(cmd.Context())
+		composeFilePath := filepath.Join(prompt.Folder, "compose.yaml")
```

Comment:
Same applies here, but during generation we know it's going to be `compose.yaml` because that's the file we generate. Regardless, would be good to not have the hardcoded name here.